recipes/referee/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set name = "Referee" %}
+{% set version = "1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gwct/referee/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 88c5639058d86dfb1c60dc490f9ac787415c901c6c5a595a65a02ed65458f0ae
+
+build:
+  noarch: generic
+  number: 0
+
+requirements:
+  run:
+    - python >=3
+    - samtools
+
+test:
+  commands:
+    - referee.py --help
+
+about:
+  home: https://github.com/gcwt/referee
+  license: GPL-3.0
```

Comment:
Can you please use a spdx identifier. Thanks.

Reply:
updated.